feature_tests/js/test/slices.mjs

```diff
@@ -9,4 +9,4 @@ test("MyString functionality", (t) => {
 test("String List", (t) => {
 	let str = MyString.newFromFirst(["This", "is", "a", "test."]);
 	t.is(str.str, "This");
-});
\ No newline at end of file
+});
```

Comment:
I don't think there's a good reason for why we do this, but tests are duplicated across all files. So we'd add:

```suggestion
});

test("Float64Vec", (t) => {
    let input = [1, 2, 3, 4, 5];
    let data = Float64Vec.newIsize(input);
    t.deepEqual(data.borrow(), input);
});
```

I think though for the long term we should really remove all the `.mjs` files that are not directly generated from typescript.